setup.py

```diff
@@ -76,7 +76,11 @@
     install_requires=[
         'google-cloud-storage>=1.35,<1.44',
         'matplotlib>=3.3,<3.6',
+        'tensorflow_datasets>=4.2,<4.5',
         'numpy~=1.16.0',
+        'onnx==1.8.1',
+        'onnxmltools==1.10.0',
+        'tf2onnx==1.9.3',
```

Comment:
I think this should also go in keras_deps? If it's only required for the keras/tensorflow examples.

Reply:
Everything using ml_interface has to be able to import onnx and onnxmltools, tf2onnx isn't actually needed anymore it seems like